lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
yeah. hmm...there are a lot of errors...but this seems to be an issue with the .eslintrc file itself too. Lot of rules having been replaced with other rules in the latest eslint.

ex:
```
    1:1    error  Rule 'no-reserved-keys' was removed and replaced by: quote-props                                                              no-reserved-keys
    1:1    error  Rule 'spaced-line-comment' was removed and replaced by: spaced-comment
```

I think we should hold off on going much further down this road (general code styling & cleanup) until the task for "[typical repository files](https://rally1.rallydev.com/#/21230556771ud/detail/task/41507057658)" has been completed. That way we know that we're dealing with the proper set of rules for our system.

The assumption being...if we have been judicious in selecting our .eslintrc file and its contents then we won't run into errors from the file itself - clouding our ability to know the exact rules our team should be using.

@conortm opinion on waiting on on code styling for that task to be done?

@e2tha-e FWIW, while I think code-styling/syntax discussions are very valuable, I'm hoping that we can get into the more substantive discussion of the system itself. I need to know if other developers see bugs, if the system works for them at all, if the way the code is put together is workable for what we're trying to do (in this PR's case, "import" styles from pattern libraries into patternlab) and/or see better ways to perform the tasks herein.